python/src/QuantileConfidence_doc.i.in

```diff
@@ -0,0 +1,440 @@
+>>> ci = algo.computeUnilateralConfidenceInterval(sample)
+"
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::QuantileConfidence::computeUnilateralRank
+"Evaluate an unilateral rank of a quantile.
+
+The lower tail rank is the largest integer :math:`k_{low}` such that:
+
+.. math::
+
+    \Prob{X_{(k)} \leq x_{\alpha}} \geq \beta.
+
+In other words, the interval :math:`\left[ X_{(k_{low})}, +\infty\right[` is a unilateral confidence
+interval for the quantile :math:`x_\alpha` with confidence :math:`\beta`.
+It is given by:
+
+.. math::
+
+    k_{low} = \overline{F}^{-1}_{n, \alpha}(\beta)
+
+where :math:`\overline{F}^{-1}_{n, \alpha}` is the complementary quantile function 
+of the binomial distribution with parameters :math:`n` and :math:`\alpha`.
+The problem has a solution only if:
+
+.. math::
+
+    1 - (1 - \alpha)^n \geq \beta.
+
+The upper tail rank is the smallest integer :math:`k_{up}` such that:
+
+.. math::
+
+    \Prob{x_{\alpha} \leq X_{(k)}} \geq \beta.
+
+In other words, the interval :math:`\left[-\infty, X_{(k_{up})}\right[` is a unilateral confidence
+interval for the quantile :math:`x_\alpha` with confidence :math:`\beta`.
+The solution is:
+
+.. math::
+
+    k_{up} = F^{-1}_{n, \alpha}(\beta)
+
+where :math:`F^{-1}_{n, \alpha}` is the quantile function of the cumulative distribution function of the 
+binomial distribution with parameters :math:`n` and :math:`\alpha`.
+The problem has a solution only if:
+
+.. math::
+
+    1 - \alpha^n \geq \beta.
+
+Parameters
+----------
+size : int
+    Sample size
+tail : bool, optional
+    Whether to return the upper or lower tail rank
+
+Returns
+-------
+rank : int
+    Rank :math:`k \in \{0, \hdots, n-1\}`
+
+Examples
+--------
+>>> import openturns as ot
+>>> import openturns.experimental as otexp
+>>> alpha = 0.05
+>>> beta = 0.95
+>>> algo = otexp.QuantileConfidence(alpha, beta)
+>>> rank = algo.computeUnilateralRank(100)
+"
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::QuantileConfidence::computeBilateralRank
+"Evaluate the bilateral rank of a quantile.
+
+The bilateral ranks :math:`k_1, k_2` are the integers minimizing the probability:
+
+.. math::
+
+    \argmin_{0 \leq k_1 \leq k_2 \leq n - 1 \, | \, \Pset(k_1 < X \leq k_2) \geq \beta} \Pset(k_1 < X \leq k_2) | 
+
+where :math:`X \sim \cB(n, \alpha)`.
+These ranks exist only if:
+
+.. math::
+
+    1 - \alpha^n - (1-\alpha)^n \ge \beta.
+
+Parameters
+----------
+size : int
+    Sample size :math:`n`
+
+Returns
+-------
+ranks : :class:`~openturns.Indices` of size 2
+    Pair of lower and upper ranks :math:`(k_1, k_2)` such that :math:`0 \leq k_1 \leq k_2 \leq n - 1`.
+
+Examples
+--------
+>>> import openturns as ot
+>>> import openturns.experimental as otexp
+>>> alpha = 0.05
+>>> beta = 0.95
+>>> algo = otexp.QuantileConfidence(alpha, beta)
+>>> k1, k2 = algo.computeBilateralRank(100)
```

Comment:
Printing the returned values of k1 and k2 will help the user to understand the feature.